test/integration/Aave.t.sol

```diff
@@ -164,9 +185,44 @@ contract AaveIntegrationTest is BaseTest
         assertEq(_manager.totalShares(lAaveToken), uint256(lAmountToManage));
     }
 
-    function testAdjustManagement_IncreaseManagementOneToken() public allNetworks allPairs
+    function testAdjustManagement_IncreaseManagementOneToken_Frozen() public allNetworks allPairs
     {
-        _increaseManagementOneToken();
+        // arrange - freeze the USDC market
+        int256 lAmountToManage = 500e6;
+        vm.prank(_aaveAdmin);
+        _poolConfigurator.setReserveFreeze(USDC, true);
+
+        // act
+        _increaseManagementOneToken(lAmountToManage);
+
+        // assert - nothing should have moved as USDC market is frozen
+        (address lAaveToken, ,) = _dataProvider.getReserveTokensAddresses(USDC);
+        assertEq(_pair.token0Managed(), 0);
+        assertEq(_pair.token1Managed(), 0);
+        assertEq(IERC20(USDC).balanceOf(address(_pair)), MINT_AMOUNT);
+        assertEq(IERC20(lAaveToken).balanceOf(address(_manager)), 0);
+        assertEq(_manager.shares(_pair, USDC), 0);
+        assertEq(_manager.totalShares(lAaveToken), 0);
+    }
+
+    function testAdjustManagement_IncreaseManagementOneToken_Paused() public allNetworks allPairs
```

Comment:
testDecrease_Paused
testDecrease_Frozen

Reply:
yeah I wrote those test cases already, haven't pushed them. You can chill first, I'll let you know when I need you to review the PR again.